Observer/Xamarin.Forms.HotReload.Observer/FileObserver.cs

```diff
@@ -61,7 +62,8 @@ private static void Run()
                 Path = path,
                 NotifyFilter = NotifyFilters.LastWrite,
                 Filter = "*.xaml",
-                EnableRaisingEvents = true
+                EnableRaisingEvents = true,
+                IncludeSubdirectories = true,
```

Comment:
remove comma please (at the end of the line)

Reply:
👌

If you put a comma in the last line, `git diff` will be a little cleaner because there are fewer changes. It is also easier to change the order of the listed property initialization code. This is because it is not necessary to modify the comma even if the last attribute initialization code is moved up. I think this is a personal style, so I will modify it as you requested!